client/webserver/site/src/js/register.js

```diff
@@ -189,6 +194,14 @@ export default class RegistrationPage extends BasePage {
     const res = await postJSON('/api/register', registration)
     loaded()
     if (!app.checkResponse(res)) {
+      // show different form with no passphrase input in case of no funds.
+      if (res.code === sendFeeErr) {
```

Comment:
If `Code` is omitted (either the backend used the old `writeAPIError`, or used the new `writeAPIErrorPOC` but with `Code=0` which `omitempty` also includes), wont this have a js error with res.code being undefined?

Reply:
Yeah, it will be undefined, which will be !== from `sendFeeErr`, so we are good. It would crash everything if res is undefined, which I believe it never is, is that right? 

Lookin into `checkResponse`, it does not check for res to be defined.